Bluejay.asm

```diff
@@ -4474,7 +4501,8 @@ ENDIF
 	subb	A, #4					; Maximum consecutive stalls
 	jnc	exit_run_mode_stall_done
 
-	call	wait100ms					; Wait for a bit between stall restarts
+	setb	Flag_Stall_Notify		; Notify motor stall
```

Comment:
Please keep the indentations consistent with the other flags

Reply:
Fixed